src/main/java/teamrtg/highlandsut/block/HighlandsBlocks.java

```diff
@@ -2,7 +2,7 @@
 
 import net.minecraft.block.Block;
 import net.minecraft.client.Minecraft;
-import net.minecraft.client.resources.model.ModelResourceLocation;
+import net.minecraft.client.renderer.block.model.ModelResourceLocation;
```

Comment:
Sorry mate :( we'll have to coordinate what we're workin on a bit better moving forward.